src/com/plasstech/lang/d2/parse/Parser.java

```diff
@@ -285,9 +285,9 @@ private StatementNode startsWithVariableStmt() {
         Token opEq = token;
         advance(); // eat the token
         VariableSetNode vsn = new VariableSetNode(variable.text(), variable.start());
-        ExprNode rhs = expr();
+        ExprNode originalRhs = expr();
         ExprNode left = new VariableNode(variable.text(), variable.start());
-        ExprNode foo = new BinOpNode(left, OP_EQ_TO_OP.get(opEq.type()), rhs);
+        ExprNode newRhs = new BinOpNode(left, OP_EQ_TO_OP.get(opEq.type()), rhs);
         return new AssignmentNode(vsn, foo);
```

Comment:
This will not compile. Please make sure you compile and, if possible, test your change before sending a PR.